README.md

```diff
@@ -1,2 +1,222 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 1- Html
+### 2- CSS
+### 3- Javascript
+### 4- React
+### 5- Redux
+### 6- Jest
+
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Html</a></li>
+    <li><a>Css</a></li>
+    <li><a>Javascript</a></li>
+    <li><a>React</a></li>
+    <li><a>Redux</a></li>
+    <li><a>Jest</a></li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Display Weather**
+- **Search for Locations**
+- **Mobile Responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### 1- Web Browser
+### 2- Code Editor
+### 3- Git
+### 4- Node.js
+### 5- NPM
+### 6- React
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run This Command:
+
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse.git
+```
+
+
+### Install
+
+Install this project with:
+
+Run This Command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+> npm run start 
+
+
+### Run tests
+
+> npm run test
+
+### Deployment
+
+> Here is the Link to this Live Project:
+
+> https://space-travelers-hub-three.vercel.app/
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdul Wahab**
+
+- GitHub: [@githubhandle](https://github.com/Abdulwhbhussain)
+- Twitter: [@twitterhandle](https://twitter.com/AbdulWhbHussain)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abdulwhbhussain)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add more pages
+- Add color theme
+- Add user friendly animation
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project Leave A Star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and The Code Reviewers of This Project for giving 
+Reviews and Feedback Regarding This Project. I would like to thank Microverse for this opportunity. Ko Kaung Myat Kyaw(Berry) for Design ideas.
+
```

Comment:
- [ ] The right author is Nelson Sakwa. Kindly edit this section to give him credit 👍 

### Instructions:
![image](https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse/assets/50990925/1e831e22-b03a-48fb-8dbb-bf0e2f124943)

Reply:
Good Done!